spec/fixtures/manifest-with-static-name.yml

```diff
@@ -4,8 +4,16 @@ name: service-ha
 instance_groups:
 - name: database
   instances: 3
+  networks:
+  - name: dynamic
+  properties:
+    network: dynamic
```

Comment:
is this right? Looks dynmic :-D

Reply:
According to other manifests it is. https://github.com/anynines/mongodb-boshrelease/blob/master/unit-tests/manifests/cluster-ssl.yml#L36